backend/src/migrations/004_comment_response_refactor.ts

```diff
@@ -0,0 +1,73 @@
+import AccessRequestModel from '../models/AccessRequest.js'
+import ReleaseModel from '../models/Release.js'
+import ResponseModel, { ResponseKind } from '../models/Response.js'
+import ReviewModel from '../models/Review.js'
+import { toEntity } from '../utils/entity.js'
+
+export async function up() {
+  // Resolve release comments
+  const releases: any = await ReleaseModel.find({})
+  for (const release of releases) {
+    release.commentIds = []
+    if (release.get('comments') !== undefined) {
+      for (const releaseComment of release.get('comments')) {
+        const newComment = new ResponseModel({
+          user: toEntity('user', releaseComment.user),
```

Comment:
The age old argument of "Should this be an entity or not".  In this case, it's _always_ going to be a user, so maybe it should just use `releaseComment.user`, instead of an entity.